src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, String fileName) {
+        String fullPath = String.valueOf(
+                Paths.get("src", "main", "java", "resources", fileName));
```

Comment:
the same. Also, check the resources directory level, the correct path is `src/main/resources/file.csv`